### PR TITLE
Added support page

### PR DESCRIPTION
**Pull request by maksimbell**
1. I have added a support.html
2 .I dont know for what i did it.

### CODE REVIEW

**Comment by outsmartu** [comment]
Wow, cool markup man!

File: pages/support.html.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Support-page</title>
+</head>
+<body>
+
```
Comment (outsmartu) [inline pages/support.html.txt:8]: Delete all empty lines and change file extension in file editor:D

Comment (maksimbell) [reply pages/support.html.txt:8]: No, ty

**Review by outsmartu — COMMENTED** [review]
Ok, not bad for first commit

**Pull request merged**